libraries/key-identifier/src/key-identifier.d.ts

```diff
@@ -0,0 +1,20 @@
+type ConstructorParams = {
+  derivationAlgorithm: 'BIP32' | 'SLIP10'
+  derivationPath: string
+  assetName?: string
+  keyType: 'legacy' | 'nacl' | 'secp2561'
+}
+
+type KeyIdentifierLike = Partial<ConstructorParams>
+
+export default class KeyIdentifier {
+  derivationAlgorithm: ConstructorParams['derivationAlgorithm']
+  derivationPath: ConstructorParams['derivationPath']
+  assetName: ConstructorParams['assetName']
+  keyType: ConstructorParams['keyType']
+
+  constructor(params: ConstructorParams)
+
+  validate(potentialKeyIdentifier: KeyIdentifierLike): boolean
+  compare(a: KeyIdentifierLike, b: KeyIdentifierLike): boolean
```

Comment:
no need to differentiate static vs instance methods?

Reply:
good catch, they do have to be differentiated, fixed in https://github.com/ExodusMovement/exodus-oss/pull/92/commits/2e50bccaa6b900524bab2e24f7d1f2055219b22b